Contributing.md

```diff
@@ -0,0 +1,16 @@
+
+### Committing a change
+
+#### Development
+
+When opening a pull request, add your changes to the "[Unreleased]" section of the [changelog](https://github.com/zendesk/active_record_shards/blob/master/Changelog.md).
+
+#### Release
+
+After merging your changes into master, cut a tag and push it immediately:
```

Comment:
I would actually prefer that people open a pull request with the version bump.

Reply:
updated the steps to the list above